app/src/main/java/com/infomaniak/mail/ui/MainViewModel.kt

```diff
@@ -520,10 +525,27 @@ class MainViewModel(application: Application) : AndroidViewModel(application) {
         }
     }
 
+    fun reportPhishing(threadUid: String, message: Message) = viewModelScope.launch(Dispatchers.IO) {
```

Comment:
Instead of doing both actions in the same function (phishing + spam), we should do like the `readAndArchive` action.